torchci/scripts/check_alerts.py

```diff
@@ -168,24 +201,43 @@ def generate_failed_job_issue(failed_jobs: List[JobStatus]) -> Any:
 
     body += "Please review the errors and revert if needed."
     issue["body"] = body
-    issue["labels"] = labels
+    issue["labels"] = [PYTORCH_ALERT_LABEL]
 
     print("Generating alerts for: ", failed_jobs)
     return issue
 
 
-def update_issue(issue: Any, issue_number: int) -> None:
+def generate_no_flaky_tests_issue() -> Any:
+    issue = {}
+    issue[
+        "title"
+    ] = f"[Pytorch][Warning] No flaky test issues has been detected in the past {FLAKY_TESTS_SEARCH_PERIOD_DAYS} days!"
+    issue[
+        "body"
+    ] = f"No issues has been filed in the past {FLAKY_TESTS_SEARCH_PERIOD_DAYS} days for the repository {REPO_OWNER}/{TEST_INFRA_REPO_NAME}. \n"
+    issue[
+        "body"
+    ] += "This can be an indication that the flaky test bot has stopped filing tests."
```

Comment:
Nit: We tend to use this format for multiline strings

```suggestion
    ] = (
    f"No issues has been filed in the past {FLAKY_TESTS_SEARCH_PERIOD_DAYS} days for the repository {REPO_OWNER}/{TEST_INFRA_REPO_NAME}. \n"
    "This can be an indication that the flaky test bot has stopped filing tests."
 )
```